protocol/block.go

```diff
@@ -134,10 +136,13 @@ func (c *Chain) SaveBlock(block *legacy.Block) error {
 }
 
 func (c *Chain) ProcessBlock(block *legacy.Block) (bool, error) {
-	if blockHash := block.Hash(); c.BlockExist(&blockHash) {
+	blockHash := block.Hash()
+	if c.BlockExist(&blockHash) {
+		log.WithFields(log.Fields{"hash": blockHash.String()}).Info("Skip process due to block already been handled")
```

Comment:
If there is only one field, you can simply use `log.WithField()` instead, e.g., 

```go
log.WithField("hash", blockHash.String()).Info("Skip process due to block already been handled")
```

Reply:
done